sdcm/cluster.py

```diff
@@ -2111,7 +2111,7 @@ def install_mgmt(self, scylla_repo, scylla_mgmt_repo, auth_token, segments_per_r
             except Exception as ex:  # pylint: disable=broad-except
                 self.log.warning(ex)
         else:
-            self.remoter.run('sudo scyllamgr_setup -y')
+            self.remoter.run('echo yes| sudo scyllamgr_setup')
```

Comment:
@yarongilor did `scyllamgr_setup` setup changed?

Reply:
@bentsi , i think it will be better to confirm this with @ShlomiBalalis since he's writing tests for the new backup feature.

Reply:
@dkropachev , you better use as it was, ```sudo scyllamgr_setup -y```

Reply:
@fgelcer why did you change it to `scyllamgr_setup -y`?

Reply:
your change broke provision test

Reply:
because manager supports it, instead of the "trick" to ```echo yes```

Reply:
which manager version does it use?
we should move every usage of manager to 2.0

Reply:
scylla_mgmt_repo: 'http://downloads.scylladb.com.s3.amazonaws.com/manager/rpm/unstable/centos/master/latest/scylla-manager.repo'

Reply:
with Scylla 3.2.rc1

Reply:
it used defaults: 
```
scylla_repo_m: 'http://repositories.scylladb.com/scylla/repo/qa-test/centos/scylladb-2019.1.repo'
scylla_mgmt_repo: 'http://downloads.scylladb.com.s3.amazonaws.com/manager/rpm/unstable/centos/master/latest/scylla-manager.repo'
```

Reply:
from the `defaults/test_default.yaml`

Reply:
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR > Cluster longevity-200gb-48h-network-master-monitor-set-4ff29c0e (AMI: ['ami-02eac2c0129f6376b'] Type: t3.large): Node setup failed: Node longevity-200gb-48h-network-master-monitor-node-4ff29c0e-1 [35.173.213.43 | 172.30.0.193] (seed: False) < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR > Cluster longevity-200gb-48h-network-master-monitor-set-4ff29c0e (AMI: ['ami-02eac2c0129f6376b'] Type: t3.large): Node setup failed: Node longevity-200gb-48h-network-master-monitor-node-4ff29c0e-1 [35.173.213.43 | 172.30.0.193] (seed: False)
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR > Traceback (most recent call last):
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >   File "/sct/sdcm/cluster.py", line 2754, in node_setup
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >     cl_inst.node_setup(node, **setup_kwargs)
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >   File "/sct/sdcm/cluster.py", line 3806, in node_setup
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >     self.install_scylla_manager(node, auth_token=self.mgmt_auth_token)
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >   File "/sct/sdcm/cluster.py", line 3814, in install_scylla_manager
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >     segments_per_repair=self.params.get('mgmt_segments_per_repair'))
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >   File "/sct/sdcm/cluster.py", line 2114, in install_mgmt
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >     self.remoter.run('sudo scyllamgr_setup -y')
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >   File "/sct/sdcm/remote.py", line 261, in run
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >     result = _run()
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >   File "/sct/sdcm/utils/common.py", line 92, in inner
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >     return func(*args, **kwargs)
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >   File "/sct/sdcm/remote.py", line 248, in _run
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >     watchers=watchers, timeout=timeout)
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >   File "<decorator-gen-3>", line 2, in run
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >   File "/usr/local/lib/python3.6/site-packages/fabric/connection.py", line 30, in opens
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >     return method(self, *args, **kwargs)
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >   File "/usr/local/lib/python3.6/site-packages/fabric/connection.py", line 721, in run
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >     return self._run(self._remote_runner(), command, **kwargs)
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >   File "/usr/local/lib/python3.6/site-packages/invoke/context.py", line 101, in _run
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >     return runner.run(command, **kwargs)
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >   File "/usr/local/lib/python3.6/site-packages/invoke/runners.py", line 291, in run
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >     return self._run_body(command, **kwargs)
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >   File "/usr/local/lib/python3.6/site-packages/invoke/runners.py", line 442, in _run_body
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR >     raise UnexpectedExit(result)
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR > invoke.exceptions.UnexpectedExit: Encountered a bad command exit code!
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR > 
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR > Command: 'sudo scyllamgr_setup -y'
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR > 
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR > Exit code: 1
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR > 
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR > Stdout:
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR > 
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR > unknown command -y
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR > 
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR > Stderr:
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR > 
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR > 
15:55:59  < t:2020-01-22 12:55:56,567 f:cluster.py      l:2756 c:sdcm.cluster         p:ERROR > 
15:55:59  < t:2020-01-22 12:55:56,587 f:tester.py       l:129  c:sdcm.tester          p:ERROR > Exception in setUp. Will call tearDown < t:2020-01-22 12:55:56,587 f:tester.py       l:129  c:sdcm.tester          p:ERROR > Exception